src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml

```diff
@@ -5,25 +5,76 @@
     </div>
 </div>
 
-<main class="govuk-main-wrapper dashboard" id="main-content" role="main">
+<section class="das-section das-section--search">
+    <div class="govuk-width-container">
+        <div class="das-dashboard-header__row">
+            <div class="das-dashboard-header__col das-dashboard-header__col--primary">
+                <form>
+                    <div class="govuk-form-group das-form-group-inline das-dashboard-search">
+                        <label for="search" class="govuk-visually-hidden">Search</label>
+                        <input class="govuk-input das-dashboard-search__input" id="search" type="search" placeholder="Search" disabled="disabled">
+                        <div class="das-form-group-inline__button-wrap">
+                            <button class="govuk-button das-dashboard-search__button" type="submit">Search</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="das-dashboard-header__col das-dashboard-header__col--secondary">
+                <h2 class="das-placeholder-tasks">Other tasks you can do</h2>
+            </div>
+        </div>
+    </div>
+</section>
+
+
+<main class="govuk-main-wrapper das-section--dashboard" id="main-content" role="main">
     <div class="govuk-width-container">
         <div class="das-panels">
-            <div class="das-panels__row">
+            <div class="das-panels__col das-panels__col--primary">
                 @Html.Action("Row1Panel1", new { model = Model.Data })
-                @Html.Action("Row1Panel2", new { model = Model.Data })
-            </div>
-            <div class="das-panels__row das-panels__row--secondary">
                 @Html.Action("Row2Panel1", new { model = Model.Data })
+            </div>
+            <div class="das-panels__col das-panels__col--secondary">
+                @Html.Action("Row1Panel2", new { model = Model.Data })
                 @Html.Action("Row2Panel2", new { model = Model.Data })
             </div>
         </div>
     </div>
 </main>
 
+<section class="das-section das-section--main das-dashboard-footer" aria-labelledby="more-title">
+    <div class="govuk-width-container">
+        <div class="govuk-grid-row">
+            <hr class="das-section-break das-section-break--l das-section-break--heavy das-section-break--gutters">
+            <div class="das-dashboard-footer__column das-dashboard-footer__column--title">
+                <h2 id="more-title" class="das-dashboard-footer__title">More on the Apprenticeships Service</h2>
+            </div>
+            <div class="das-dashboard-footer__column">
+                <h3 class="govuk-heading-m das-dashboard-footer__sub-title">Most active</h3>
+                <ul class="govuk-list das-list--featured">
+                    <li><a href="@("/commitments/accounts/"+Model.Data.HashedAccountId+"/apprentices/home")" class="govuk-link govuk-link--no-visited-state">Add an apprentice</a></li>
```

Comment:
top two links need to have the commitments base url prepended and the 3rd one needs Provider relationships base url

Reply:
Done